app/models/service_update.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class ServiceUpdate
```

Comment:
The name update will probably be confusing down the line. Can you use a more specific name like user feed(probably ask product people)